zenoh/src/publication.rs

```diff
@@ -241,6 +242,7 @@ impl<'a> Publisher<'a> {
 
     /// Restrict the matching subscribers that will receive the published data
     /// to the ones that have the given [`Locality`](crate::prelude::Locality).
+    #[cfg(feature = "unstable")]
```

Comment:
`Locality` should be only on subscriber and queryable side for the time being at least.